MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m

```diff
@@ -9,6 +9,16 @@ @interface MatchmakingViewController ()<UITableViewDataSource, UITableViewDelega
     
 @implementation MatchmakingViewController
 
++ (instancetype)sharedMatchmaking {
+    static MatchmakingViewController * MatchVC = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        MatchVC = [[self alloc] init];
```

Comment:
Did you want to instantiate it this way, or using a storyboard like we did the other day for the LoginViewController?